### PR TITLE
Code improvement

### PR DESCRIPTION
**Pull request by B0pol**
- [i] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [ ] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).
- [ ] I agree to create a pull request for [NewPipe](https://github.com/TeamNewPipe/NewPipe) as soon as possible to make it compatible with the changed API.

No need

### CODE REVIEW

**Review by TobiGr — APPROVED** [review]

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsInfoItemExtractor.java [hunk 1]
```diff
@@ -46,7 +46,7 @@ public String getName() throws ParsingException {
         try {
             return getTextFromObject(JsonUtils.getObject(json, "authorText"));
         } catch (Exception e) {
-            return "";
+            return EMPTY_STRING;
```
Comment (mhmdanas) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsInfoItemExtractor.java:49]: What is the purpose of this change?

Comment (B0pol) [reply extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsInfoItemExtractor.java:49]: Use only once the same variable

**Comment by TobiGr** [comment]
needs a rebase
